base/hourai/bot/extensions/verification/rejectors.py

```diff
@@ -36,8 +36,9 @@ async def verify_member(self, ctx):
         field_value = self.subfield(ctx.member)
         for filter_name, regex in member_names.items():
             if re.search(regex, field_value):
-                ctx.add_rejection_reason(
```

Comment:
This is better to check if the member's is identical to the user being verified.

Reply:
Fixed the issue, now performs that check.